dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/security/impl/AbstractAuthenticator.java

```diff
@@ -41,7 +41,7 @@ public abstract class AbstractAuthenticator implements Authenticator {
     private static final Logger logger = LoggerFactory.getLogger(AbstractAuthenticator.class);
 
     @Autowired
-    private UsersService userService;
+    protected UsersService userService;
```

Comment:
What about renaming the Class `UsersService` with `UserService`? Looks kinda wired.